regtest/aports-setup.sh

```diff
@@ -450,6 +470,8 @@ fetch-all() {
   clone-aports
   clone-aci
   checkout-stable
+  patch-aports
```

Comment:
Oh there is actually a function `patch-yash-to-disable-tests`, which is done AFTER we copy `../alpinelinux/aports` INTO the chroot

We should probably settle on one or the other

I guess I didn't want to mess up the local copy, so I patched the one in the chroot

Reply:
re-running `fetch-all` on changes to this branch works for me, but `prepare-all` is not idempotent: `ERROR: Directory oils-for-linux/oils/_chroot/aports-build should be empty` and I didn't want to remove the chroot every time.

One can always restore the local copy imo - it's a git repo, just `git restore .` works

Reply:
OK, yeah I guess it's better to patch without root access, which `patch-yash` needs